Sources/FluentKit/Enum/EnumProperty.swift

```diff
@@ -0,0 +1,72 @@
+extension Fields {
+    public typealias Enum<Value> = EnumProperty<Self, Value>
+        where Value: Codable,
+            Value: RawRepresentable,
+            Value.RawValue == String
```

Comment:
Could we constrain `Value.RawValue` to `LosslessStringConvertible` instead of `String`? That way it would at least also support integer type backed enums.

Reply:
Native fluent enums can only be used with string-backed enums where the strings match the cases you add to the `database.enum("enumName")` builder. This is because enums must be passed by name in queries (they aren't bound). For example:
```sql
SELECT * FROM planets WHERE name = ? AND type = 'smallRocky'
```